src/routers/index.js

```diff
@@ -6,15 +6,17 @@ import checkFields from '../middlewares/auth/loginValidator';
 import socialRedirect from '../controllers/authentication/socialRedirect';
 import { login, createUser } from '../controllers/authentication/user';
 import checkBody from '../middlewares/signUpValidator';
+import likeArticle from '../controllers/like';
+import Authenticator from '../middlewares/authenticator';
 
 
 const router = Router();
 
-router.get('/', (req, res) => {
-  res.status(200).json({
-    message: 'welcome to the base url',
-  });
-});
+router.get('/', (req, res) => res.status(200).json({
+  message: 'Welcome to the Authors Haven API',
```

Comment:
You might want to add the response format here too

Reply:
> You might want to add the response format here too

Thanks for pointing that out @Raymond-Osy. It's been updated in my PR